dali/python/nvidia/dali/_multiproc/pool.py

```diff
@@ -375,7 +386,7 @@ def from_contexts(cls, contexts : List[CallbackContext], num_workers, start_meth
                 if worker_context.dedicated_task_queue is not None:
                     worker_context.dedicated_task_queue.close_handle()
             return instance
-        except:
+        except (OSError, ValueError):
```

Comment:
This is a cleanup section - silence the warning instead:
```suggestion
        except:  # noqa: E722
```

Reply:
Done